assets/configuration/core_configuration.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "0.0.1",
+  "version": "0.0.2",
```

Comment:
Should this version number sync up with the [tags](https://github.com/ns8inc/protect-sdk-php/tags)?

Reply:
@davehensleyns8  I thought we intended to keep this separate from the tag earlier on and use it for programmatic version checks which is why we kept it at `0.0.1`.

Reply:
Ok, works for me. 👍